swagger.yml

```
@@ -834,6 +834,23 @@ paths:
               $ref: '#/definitions/dnsZone'
         default:
           $ref: '#/responses/error'
+  /sites/{site_id}/rollback:
+    parameters:
+      - name: site_id
+        type: string
+        in: path
+        required: true
+    post:
+      summary: Rollback production deploys
```

The deploy that's rolled back is singular, so I'd suggest changing this summary wording:
```suggestion
      summary: Rollback production deploy
```